client/my-sites/comments/comment/index.jsx

```diff
@@ -0,0 +1,91 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ReactDom from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CommentHeader from './comment-header';
+
+export class Comment extends Component {
+	static propTypes = {
+		commentId: PropTypes.number,
+		isBulkMode: PropTypes.bool,
+		isLoading: PropTypes.bool,
+		isSelected: PropTypes.bool,
+		siteId: PropTypes.number,
+	};
+
+	static defaultProps = {
+		isBulkMode: false,
+		isLoading: false,
+		isSelected: false,
+	};
+
+	state = {
+		isEditMode: false,
+		isExpanded: false,
+	};
+
+	storeCardRef = card => ( this.commentCard = card );
+
+	keyDownHandler = event => {
+		const commentHasFocus =
+			document &&
+			this.commentCard &&
+			document.activeElement === ReactDom.findDOMNode( this.commentCard );
```

Comment:
Interesting, did we need to do this earlier?

Reply:
Yup, this is one of the things I didn't dare changing. Testing accessibility and keyboard navigation = 😱 

Though, it originated here: https://github.com/Automattic/wp-calypso/pull/16339